scripts/table_updates/update_data_table.py

```diff
@@ -24,307 +24,438 @@
 
 from utilities import *
 
-TABLE_INFO = {"primary": ('syn23285911',"table_type='data'"),
-              "irr": ('syn21446696',"table_type='data' and double_curated is true")}
+TABLE_INFO = {
+    "primary": ("syn23285911", "table_type='data'"),
+    "irr": ("syn21446696", "table_type='data' and double_curated is true"),
+}
+
+
+def get_main_genie_clinical_sample_file(
+    syn: synapseclient.Synapse, release: str, release_files_table_synid: str
+) -> pandas.DataFrame:
+    """This retrieves the main genie clinical sample file from consortium release
+
+    Args:
+        release (str): release version to pull from for main genie
+        release_files_table_synid (str): synapse id of the data relese files table
+        from main genie
+
+    Returns:
+        pandas.DataFrame: the read in clinical file as dataframe
+    """
+    release_files = syn.tableQuery(
+        f"SELECT * FROM {release_files_table_synid}"
+    ).asDataFrame()
+    #assert_frame_equal(release_files, release_files)
+    clinical_link_synid = release_files[
+        (release_files["release"] == release)
+        & (release_files["name"] == "data_clinical_sample.txt")
+    ]["fileSynId"].values[0]
+    clinical_link_ent = syn.get(clinical_link_synid)
```

Comment:
Use `followLink=True` parameter to download the linked file.

Reply:
This way you don't need to specify the `version` parameter as well?

Reply:
That's right, it will download the correct version